apps/src/templates/sectionAssessments/MultipleChoiceAssessmentsOverviewTable.jsx

```diff
@@ -45,6 +56,7 @@ const answerColumnsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex
   return (
       <MultipleChoiceAnswerCell
         id={rowData.id}
+        opacity={calculateOpacity(percentValue)}
```

Comment:
since you are already passing `percentValue` into `MultipleChoiceAnswerCell`, I'd let `MultipleChoiceAnswerCell` handle opacity calculations, that way you don't have to pass in another prop. (just move your `calculateOpacity` func to `MultipleChoiceAnswerCell`)